crates/schema-impl/src/schemas_impl.rs

```diff
@@ -872,6 +902,7 @@ mod tests {
 
         schemas.assert_service_revision(mocks::GREETER_SERVICE_NAME, 1);
         schemas.assert_resolves_endpoint(mocks::GREETER_SERVICE_NAME, endpoint.id());
+        // assert_eq!(schemas.list_services().first().unwrap().methods.len(), 1);
```

Comment:
I think we can now enable this, since we're filtering only for the requested methods.

```suggestion
        assert_eq!(schemas.list_services().first().unwrap().methods.len(), 1);
```

Reply:
Will also apply this to next PR